doc/lua_api.txt

```diff
@@ -948,7 +948,8 @@ Node paramtypes
 The functions of `param1` and `param2` are determined by certain fields in the
 node definition.
 
-`param1` is reserved for the engine when `paramtype != "none"`:
+Meaning of `param1` is determined by `paramtype` in node definition.
```

Comment:
"The meaning"...

Reply:
"The function of ..." would be better as it is consistent with the lines above.